packages/waku/src/router/fs-router.ts

```diff
@@ -149,6 +156,15 @@ export function unstable_fsRouter(
             render: 'static',
             ...config,
           });
+        } else if (pathItems.at(-1)?.startsWith('_part')) {
+          const order = parseInt(pathItems.at(-1)!.split('-')[0]!.substring(5));
+          createPagePart({
+            path,
+            component: mod.default,
+            render: 'static',
```

Comment:
It feels like we should make "render: 'static'" by default for `createPage`.

Reply:
So nextjs actually used this default and had a lot of user issues with it. Next 15 made pages dynamic by default (by popular demand): [link to announcement](https://arc.net/l/quote/imbnghlr)

I should make the pagePart default render be dynamic. It matches what seems to be intuitive for users.

Maybe we should make layouts dynamic by default as well if we want consistency though?

Reply:
> Maybe we should make layouts dynamic by default as well if we want consistency though?

That sounds better for consistency.

Though, this is still a simple rule:
- layout, root: static by default
- page, page parts: dynamic by default

Hm, I may prefer everything dynamic or static by default. Let's discuss with @sandren after we finish v0.22.